python/rikai/spark/sql/codegen/base.py

```diff
@@ -143,9 +143,10 @@ def pre_processing(self) -> Optional[Callable]:
         if (
             "transforms" not in self._spec
             or "pre" not in self._spec["transforms"]
+            or self._spec["transforms"]["pre"] is None
         ):
             # Passthrough
-            return lambda x: x
+            return None
```

Comment:
Why returns None here? Will it break some caller which did not check the return?

IIRC, using `lambda x: x` was to make the absent of pre/post processor an non-ops in inference runner.

Reply:
https://github.com/eto-ai/rikai/blob/c20aead79bd2525ce9d75140cd42871765abbce8/python/rikai/spark/sql/codegen/pytorch.py#L77-L78

> Why returns None here?

For performance concerns.

Reply:
Reverted to identity function.